types/rpi-gpio/index.d.ts

```diff
@@ -0,0 +1,136 @@
+// Type definitions for rpi-gpio 2.1
+// Project: https://github.com/JamesBarwell/rpi-gpio.js#readme
+// Definitions by: Giles Roadnight <https://github.com/Roaders>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.0
+
+/// <reference types="node" />
+import { EventEmitter } from 'events';
+
+type MODE = 'mode_rpi' | 'mode_bcm';
+type DIR = 'in' | 'out' | 'low' | 'high';
+type EDGE = 'none' | 'rising' | 'falling' | 'both';
+type ValueCallback<T> = (err?: Error | null, value?: T) => void;
+type ErrorCallback = (err?: Error | null) => void;
+
+declare class Gpio extends EventEmitter {
+    constructor();
+
+    readonly DIR_IN: DIR;
+    readonly DIR_OUT: DIR;
+    readonly DIR_LOW: DIR;
+    readonly DIR_HIGH: DIR;
+    readonly MODE_RPI: MODE;
+    readonly MODE_BCM: MODE;
+    readonly EDGE_NONE: EDGE;
+    readonly EDGE_RISING: EDGE;
+    readonly EDGE_FALLING: EDGE;
+    readonly EDGE_BOTH: EDGE;
+    /**
+     * Set pin reference mode. Defaults to 'mode_rpi'.
+     *
+     * @param mode Pin reference mode, 'mode_rpi' or 'mode_bcm'
+     */
+    setMode(mode: MODE): void;
+    /**
+     * Setup a channel for use as an input or output
+     *
+     * @param  channel   Reference to the pin in the current mode's schema
+     * @param _direction The pin direction, either 'in' or 'out'
```

Comment:
out of the curiosity, why this convention with lodash? it's very uncommon

Reply:
this was internal implementation when I was working on a TS version of this library (the basis of these types). I'll remove